specification/iotoperations/IoTOperations.Management/models/dataflows/dataflowEndpoints.tsp

```diff
@@ -296,6 +333,11 @@ model DataflowEndpointLocalStorage {
 
 @doc("Broker endpoint properties")
 model DataflowEndpointMqtt {
+  @doc("authentication properties. DEFAULT: kubernetes.audience=aio-mq-internal. NOTE - Enum field only property is allowed")
+  authentication?: DataflowEndpointMqttAuthentication = #{
```

Comment:
Sorry, I think with this https://msazure.visualstudio.com/One/_workitems/edit/28622511 it means that `authentication` should always be required. Or, in the case for local storage, not present